addons/netfox.internals/logger.gd

```diff
@@ -70,7 +70,10 @@ func _check_log_level(level: int) -> bool:
 
 func _format_text(text: String, level: int) -> String:
 	level = clampi(level, LOG_MIN, LOG_MAX)
-	return "[%s][%s::%s] %s" % [level_prefixes[level], module,name, text]
+	var peer_id = NetworkEvents.multiplayer.get_unique_id()
+	var tick = NetworkTime.tick
```

Comment:
statically typed for both variables gives greater performance

Reply:
Actually, the variable tick is needless, why not use directly `NetworkTime.tick`?

Reply:
Switched `=` to `:=` so type is inferred. I'll keep the variables in the function, as they're only needed for the function. 
I've extracted `tick` to a separate variable as well, so that the log line doesn't get too long.

Reply:
Alright. 

Btw `:=` is bad practice, because it hides the type from the user, just like var without specifying the type. So it is unclear if a variable is integer/float etc unless you search for return type in some previous line, by pressing ctrl to that function. An example I saw today in this codebase is the variable "port" which is in some places `String` in others `int`, so with `:=` it can still cause bugs if that variable is used further.

Reply:
Yeah, imo at this point it comes down to code style and conventions.

I dislike typing variables inside functions - at that point you want to write logic, instead of constricting declarations. Dynamic typing keeps things malleable and reduces friction when iterating on code. Members are always statically typed, so whenever you refer to them, you can be sure to have a fixed type.

I'm good with experimenting with `:=`, since it doesn't require explicit typing, but has the potential to improve performance slightly due to the inferred type.

As for figuring out types, I hope that code is legible enough that people can figure out the types given the context :slightly_smiling_face: That, and I see more importance in semantics rather than syntactics.